exercises/nodejs/src/game.js

```diff
@@ -1,154 +1,159 @@
 import generator from 'random-seed'
+import { Player } from './player'
 
 var Game = function () {
-  var players = new Array()
-  var places = new Array(6)
-  var purses = new Array(6)
-  var inPenaltyBox = new Array(6)
+  const MAX_PLAYERS = 6
+  const winningGoldCoins = 6
+  var players = []
+  var board = new Array(MAX_PLAYERS)
 
-  var popQuestions = new Array()
-  var scienceQuestions = new Array()
-  var sportsQuestions = new Array()
-  var rockQuestions = new Array()
+  var popQuestions = []
+
+  var scienceQuestions = []
+  var sportsQuestions = []
+  var rockQuestions = []
 
   var currentPlayer = 0
-  var isGettingOutOfPenaltyBox = false
 
-  var didPlayerWin = function () {
-    return !(purses[currentPlayer] == 6)
+  var isWinner = function () {
```

Comment:
```suggestion
  var shouldContinueGame = function () {
```
The variable name might be more descriptive if it's something like shouldContinueGame, as the player only wins and the game ends if purse == winningGoldCoins